registry/handlers/images.go

```diff
@@ -183,6 +223,8 @@ func (imh *imageManifestHandler) GetImageManifest(w http.ResponseWriter, r *http
 	w.Header().Set("Docker-Content-Digest", imh.Digest.String())
 	w.Header().Set("Etag", fmt.Sprintf(`"%s"`, imh.Digest))
 	w.Write(p)
+
+	fmt.Printf("\n\nThey succeeded!\n\n\n")
```

Comment:
This shouldn't be here...without `🎉`, maybe `fmt.Printf("\n\nThey succeeded! 🎉🎉🎉🎉\n\n\n")`